<?xml version="1.0"?>
<launch>
  <arg name="gui"        default="true"/>
<!--  <arg name="world_file" default="/home/sergi/catkin_ws/src/costmap_merge/worlds/world1.world"/>-->
  <arg name="world_file" default="$(env TURTLEBOT_GAZEBO_WORLD_FILE)"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="use_sim_time" value="true"/>
    <arg name="debug" value="false"/>
    <arg name="gui" value="$(arg gui)" />
    <arg name="world_name" value="$(arg world_file)"/>
  </include>

  <node name="simulation_launcher" pkg="costmap_merge" type="simulation_launcher.py" output="screen">
    <rosparam file="$(find costmap_merge)/config/simulation.yaml" command="load"/>
  </node>

</launch>
